Makefile

```diff
@@ -34,6 +37,15 @@ static-check:
 	./scripts/lint-go.sh
 	./scripts/lint-text.sh
 
+.PHONY: cephcsi
+cephcsi:
+	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
+	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o  _output/cephcsi ./cmd/
+
+image-cephcsi:
+	cp deploy/cephcsi/image/Dockerfile _output
+	podman build -t $(CSI_IMAGE_NAME):$(CSI_IMAGE_VERSION) _output
```

Comment:
won't podman be installed first (if not present)?

Reply:
why podman? IMO most of the devs use docker. we should give an option to use other build tools

Reply:
I left that in by accident because I use podman and needed to test my changes and PR #260 has not been merged. It was not intentional. I will change it to docker next time I push unless that #260 is available to use then.